bifrost.go

```diff
@@ -191,3 +194,35 @@ func newSimpleStorageService(g *BridgeConfig) (RainbowBridge, error) {
 		Client:        client,
 	}, nil
 }
+
+func newGoogleDriveStoage(g *BridgeConfig) (rainbowBridge, error) {
+	var client *http.Client
+	var token oauth2.Token{}
+	var err error
+	if g.CredentialsFile != nil {
+		f, err := os..Open(g.CredentialsFile)
+		if err != nil {
+			return nil, &BifrostError{
+				Err: err,
+				ErrorCode: errors.ErrUnauthorized,
+			}
+		}
+
+		defer f.Close()
+		err = json.NewDecoder(f).Decode(tok)
+		if err != nil {
+			return nil, &BifrostError{
+				Err: err,
+				ErrorCode: errors.ErrUnauthorized,
+			}
+		}
+	}
+
+	client = config.Client(ctx, tok)
+	return &gdrive.GoogleDriveStorage{
```

Comment:
I can't fully pinpoint but I think, some other configs are missing in the return value here.